src/commands/publish/mod.rs

```diff
@@ -24,22 +24,10 @@ pub fn publish(
     target: &mut Target,
     verbose: bool,
 ) -> Result<(), failure::Error> {
-    let msg = match &target.route {
-        Some(route) => &route,
-        None => "workers_dev",
-    };
-
-    log::info!("{}", msg);
-
     validate_target_required_fields_present(target)?;
     validate_worker_name(&target.name)?;
 
     if let Some(site_config) = target.site.clone() {
-        if let Some(route) = &target.route {
-            if !route.ends_with("*") {
-                message::warn(&format!("The route in your wrangler.toml should have a trailing * to apply the Worker on every path, otherwise your site will not behave as expected.\nroute = {}*", route));
-            }
-        }
```

Comment:
i moved this down into the deploy routes function, since it's only relevant to that path